fixtures/showtec/led-blinder-2-cob.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Blinder 2 COB",
+  "categories": ["Blinder", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Eric Bessot", "Luke Nelson"],
+    "createDate": "2023-04-28",
+    "lastModifyDate": "2023-04-28"
+  },
+  "links": {
+    "manual": [
+      "https://www.woodbrass.com/images/woodbrass/SHOWTEC+LED+BLINDER+2+50W.PDF"
+    ]
+  },
+  "physical": {
+    "dimensions": [138, 393, 185],
+    "weight": 4.14,
+    "power": 110,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 3200
+    },
+    "lens": {
+      "degreesMinMax": [35, 35]
+    }
+  },
+  "availableChannels": {
+    "Dimmer intensity (Left COB LED)": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer intensity (Right COB LED)": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Master Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
```

Comment:
I'd rename these channels to "Dimmer Left", "Dimmer Right" and "Master Dimmer", and use a matrix.

Reply:
Hopefully my changes are ok. I've used [this fixture](https://github.com/OpenLightingProject/open-fixture-library/blob/32fbda0e8af982a454221d08beb68e245b46b437/fixtures/showtec/sunstrip-active-mkii.json) as a reference.